sequential/backend/seq2pat.cpp

```diff
@@ -30,49 +30,67 @@ namespace patterns {
     std::vector< std::vector<int> > Seq2pat::mine()
     {
         // Queue of MDD nodes
-        std::vector<Pattern*> mdd_q;
```

Comment:
can you add a comment here on the idea here? Is it to hold onto a single copy of these objects once mine() is called. And then I assume the API changes below is to pass in these single global objects?

Reply:
Yes, this is to have a single hold of the MDD nodes, then we may remove the global variables in `Build_MDD()` to hold the mdd nodes.